src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -93,6 +95,32 @@ object DUOS {
     }
   }
 
+
+  case class StructuredDataRequest(generalResearchUse: Boolean,
+                                   healthMedicalBiomedicalUseOnly: Boolean,
+                                   diseaseUseOnly: Array[Int],
+                                   commercialUseProhibited: Boolean,
+                                   forProfitUseProhibited: Boolean,
+                                   methodsResearchProhibited: Boolean,
+                                   aggregateLevelDataProhibited: Boolean,
+                                   controlsUseProhibited: Boolean,
+                                   genderUseOnly: String,
+                                   pediatricResearchOnly: Boolean,
+                                   IRB: Boolean,
+                                   prefix: String)
+
+  case class StructuredDataResponse(consentCodes: Array[String],
+                                    dulvn: Double, //what the heck is this?
+                                    prefix: String,
+                                    structuredUseRestriction: Map[AttributeName, Attribute]) {
+    def formatWithPrefix(): Map[String, JsValue] = {
+      implicit val impAttributeFormat = new AttributeFormat with PlainArrayAttributeListSerializer
+      Map(prefix + "consentCodes" -> consentCodes.toJson,
+        prefix + "dulvn" -> dulvn.toJson,
+        prefix + "structuredUseRestriction" -> structuredUseRestriction.toJson)
```

Comment:
Currently, this is the simplest implementation of adding a prefix to each field. There are a couple of other options though: 

(a) Don't use a case class - instead of creating a `StructuredDataResponse`, just turn it into a Map[String, JsValue] directly

(b) Write function that takes in any object (Product) and a prefix (String) and turns it into a Map[String, JsValue]. The problem is that .toJson get weird when you apply it to something of type Any. I sort of started going down that path - (see comment below in ModelJsonProtocol.scala) but the turning Any to json got a bit hairy.